docs/en/CSMM/custom-commands.md

```diff
@@ -239,13 +239,13 @@ You will also need 2 arguments
 Additionally you can send yourself a pm saying who you paid to and how much by adding this line to the end
 
 ```
-pm ${player.steamId} "You have just sent ${amount} to ${steam}"
+pm ${player.entityId} "You have just sent ${amount} to ${steam}"
 ```
 
 The final command is
 
 ```
-addCurrency(${steam},${amount});pm ${player.steamId} "You have just sent ${amount} to ${steam}"
```

Comment:
Since this is the `addCurrency` CSMM function, it can still handle steamId like this. It's only for the game commands that we now need the Steam_ prefix on steam IDs